spyglass/src/main/java/com/linkedin/android/spyglass/tokenization/impl/WordTokenizerConfig.java

```diff
@@ -22,6 +22,7 @@
  */
 public class WordTokenizerConfig {
 
+    @NonNull
```

Comment:
Can we add annotations for variables on the same line as the variable?

Reply:
Fixed.